packages/flame_oxygen/example/lib/main.dart

```diff
@@ -0,0 +1,51 @@
+import 'dart:math';
+
+import 'package:flame/game.dart';
+import 'package:flame_oxygen/flame_oxygen.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+import 'component/timer_component.dart';
+import 'component/velocity_component.dart';
+import 'system/debug_system.dart';
+import 'system/kawabunga_system.dart';
+import 'system/move_system.dart';
+import 'system/sprite_system.dart';
+
+void main() {
+  runApp(GameWidget(game: ExampleGame()));
+}
+
+class ExampleGame extends OxygenGame with FPSCounter {
+  @override
+  Future<void> init() async {
+    if (kDebugMode) {
```

Comment:
https://dart.dev/guides/language/effective-dart/style#dont-use-prefix-letters

Reply:
Uh kDebugMode is from Flutter?

Reply:
It is also on your guidelines to use it.

Reply:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#begin-global-constant-names-with-prefix-k